src/test/java/http/HttpSessionImplTest.java

```diff
@@ -0,0 +1,46 @@
+package http;
+
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author KingCjy
+ */
+public class HttpSessionImplTest {
+
+    @Test
+    public void initTest() {
+        HttpSessionManager sessionManager = new HttpSessionManager();
+
+        HttpSession httpSession = sessionManager.createSession();
+        assertThat(httpSession.getId()).isNotNull();
+        assertThat(sessionManager.getSession(httpSession.getId())).isEqualTo(httpSession);
+    }
+
+    @Test
+    public void attributesTest() {
+        HttpSessionImpl httpSession = new HttpSessionImpl();
+
+        httpSession.setAttribute("logined", true);
+        httpSession.setAttribute("logined2", true);
+
+        httpSession.removeAttribute("logined2");
+        assertThat(httpSession.getAttribute("logined")).isEqualTo(true);
+        assertThat(httpSession.getAttribute("logined2")).isNull();;
+    }
+
+    @Test
+    public void invalidateTest() {
+        HttpSessionImpl httpSession = new HttpSessionImpl();
```

Comment:
인터페이스와 구현체를 분리하여 구현하신 이유가 있나요?
만약에 의도하신 부분이라면 이 부분에서는 아래처럼 사용하면 어떨까요?
변수를 선언할 땐 구현체 보다는 인터페이스로 해주는 것이 인터페이스를 분리하신 장점을 활용하실 수 있을 것 같네요 ^^

```suggestion
        HttpSession httpSession = new HttpSessionImpl();
```

Reply:
처음에는 세션 인터페이스를 만들어두면 나중에 Redis 세션등을 추가할때 편할거라고 생각했었는데 생각만하고 구현을 하다말았네요.. 세션 인터페이스는 삭제하는게 나은것 같습니다.